src/app/mork/celery/tasks/sarbacane.py

```diff
@@ -99,7 +99,7 @@ def _delete_contact(client: httpx.Client, endpoint: str, email: str):
         response = client.delete(f"{endpoint}?email={email}")
         response.raise_for_status()
     except httpx.HTTPStatusError as exc:
-        if exc.response.status_code == httpx.codes.NOT_FOUND:
+        if exc.response.status_code == httpx.codes.INTERNAL_SERVER_ERROR:
             logger.info(f"User not found at {endpoint}")
         else:
             msg = f"Failed to delete user contact at {endpoint}"
```

Comment:
Maybe we should check the response message ?
```suggestion
        if exc.response.data.message == "No contacts versions to delete":
            logger.info(f"User not found at {endpoint}")
        else:
            msg = f"Failed to delete user contact at {endpoint}"
```
WDYT ?

Reply:
Yes, you're right!